examples/todoapp/test/lib_router/blocs/router_bloc_test.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:rx_bloc_test/rx_bloc_test.dart';
+import 'package:todoapp/base/common_blocs/coordinator_bloc.dart';
+import 'package:todoapp/lib_permissions/services/permissions_service.dart';
+import 'package:todoapp/lib_router/blocs/router_bloc.dart';
+import 'package:todoapp/lib_router/router.dart';
+
+import '../../base/common_blocs/coordinator_bloc_mock.dart';
+import '../../stubs.dart';
+import 'router_bloc_test.mocks.dart';
+
+@GenerateMocks([
```

Comment:
Please create a ticket for adding this test to the rx_bloc_cli as well

@DDavidPrime, @nikolay-vasilev-prime

Reply:
https://github.com/Prime-Holding/rx_bloc/issues/726 it's created!